geode-old-client-support/src/main/java/com/gemstone/gemfire/OldClientSupportProvider.java

```diff
@@ -121,11 +123,21 @@ public Throwable getThrowable(Throwable theThrowable, KnownVersion clientVersion
     if (theThrowable == null) {
       return theThrowable;
     }
+
+    String className = theThrowable.getClass().getName();
+
+    // GEODE-9452, backward compatibility for authentication expiration
+    if (clientVersion.isOlderThan(KnownVersion.GEODE_1_14_0)) {
```

Comment:
The current version in develop is 1.15. Unless this is back ported in the future to older versions then 1.15 should be used in this check.